test-suite/shared/src/test/scala/org/scalajs/testsuite/scalalib/RangesTest.scala

```diff
@@ -23,21 +23,21 @@ import org.scalajs.testsuite.utils.Platform._
 
 class RangesTest {
 
-  @Test def Iterable_range_should_not_emit_dce_warnings_issue_650(): Unit = {
+  @Test def iterableRangeDoesNotEmitDceWarnings_Issue650(): Unit = {
```

Comment:
Another test name that was wrong before:
```suggestion
  @Test def iterableRangeLinks_Issue650(): Unit = {
```
There has been no such thing as "DCE warnings" since Scala.js 0.6.0 ^^